engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/util/context/FlinkContextInitializer.scala

```diff
@@ -0,0 +1,36 @@
+package pl.touk.nussknacker.engine.flink.util.context
+
+import org.apache.flink.api.common.functions.MapFunction
+import pl.touk.nussknacker.engine.api.Context
+import pl.touk.nussknacker.engine.api.context.ProcessCompilationError.NodeId
+import pl.touk.nussknacker.engine.api.context.ValidationContext
+import pl.touk.nussknacker.engine.api.typed.typing
+
+/**
+  * FlinkContextInitializer provides definition of Context returned within DataStream by FlinkSource
+  * and map function that transforms raw input to Context.
+  *
+  * @tparam T - type of event that is generated by raw source.
+  */
+abstract class FlinkContextInitializer[T] extends Serializable {
```

Comment:
IMO those two classes should be in the same file/package as `FlinkIntermediateRawSource` because they are connected. Reading docs you don't know here what is "raw source".

Reply:
Fixed. Moved to `FlinkSource` and `FlinkIntermediateRawSource` file.

Reply:
I see that not exaclty. Have you pushed everything? Because I still see both `FlinkIntermediateRawSource` and `FlinkContextInitializer` in `FlinkSource.scala`. Can you extract them to separate `FlinkIntermediateRawSource` file like you wrote in comment?

Reply:
Fixed. Extracted both to separate file.